packages/core/scripts/EmergencyShutdown.js

```diff
@@ -25,7 +23,9 @@ async function run(account, derivative) {
 
 async function runScript(callback) {
   const account = (await web3.eth.getAccounts())[0];
-  await run(account, argv.derivative);
+  const adminAbi = artifacts.require("FinancialContractsAdmin").abi;
```

Comment:
why pass these in like this rather than just keeping them as consts at the top of the file? just trying to understand the general pattern for future files/changes.

Reply:
This is just to both keep the script truffle compatible (we haven't moved the scripts over yet), but still allow the function to be tested by the hardhat tests. This will likely be rewritten when we move the scripts over.